lib/ibm_watson/visual_recognition_v3.rb

```diff
@@ -356,7 +354,7 @@ def get_classifier(classifier_id:)
     end
 
     ##
-    # @!method update_classifier(classifier_id:, **args)
+    # @!method update_classifier(classifier_id:, positive_examples: {}, negative_examples: nil, positive_examples_filename: {}, negative_examples_filename: nil)
```

Comment:
`{}` and `nil`

Reply:
for the update_classifier operation, the "positive_examples" parameter is not required, hence the use of {} as the default value.

Reply:
cool! makes sense